Outlining the Secure Supply Chain Use Case

magowan opened the pull request:
Use Case section of roadmap.md externally links to a spreadsheet. For Secure Supply Chain this removes the need to view the external spreadsheet to understand the Use Case

fitzthum reviewed (commented):
Best to put this [here](https://confidentialcontainers.org/docs/use-cases/supply-chain/) I think

bpradipt commented on use-case-supply-chain.md:
```diff
@@ -0,0 +1,44 @@
+# Secure Supply Chain
```
nit: Secure Supply Chain (Trusted Pipeline)

bpradipt commented on use-case-supply-chain.md:
```diff
@@ -0,0 +1,44 @@
+- Confidential Computing requires a way to verify the OCI Images being used. (Signatures, Encrypted Layers etc.)
+  - Is this the OCI Image I am looking for?
+
+Being able to be able to verify the image is not sufficient for Confidential Computing if we do not trust the environment in which the evidence for verification was created.
```
```Suggestion
Being able to verify the image is not sufficient for Confidential Computing if we do not trust the environment in which the evidence for verification was created.
```

bpradipt commented:
> Best to put this [here](https://confidentialcontainers.org/docs/use-cases/supply-chain/) I think

Yeah. As we move all docs to the website, I think this should also move there. May be after reviews you can directly use the updated content and create a PR against the website repo ?

magowan commented:
Yes Once I address the comments I'll move the PR to the website repo and then come back and tidy this repo up to point there

magowan commented:
OK I have moved this PR across to the website repo here -> https://github.com/confidential-containers/confidentialcontainers.org/pull/82
Addressed your comments in the process @bpradipt 

Please review the other PR and I will close this without merge.

magowan commented:
Recreated against website repo , closing without merge
